addons/mail/wizard/mail_activity_schedule.py

```diff
@@ -340,50 +359,45 @@ def _plan_filter_activity_templates_to_schedule(self):
 
     def _get_summary_lines(self, templates):
         self.ensure_one()
-        summaries = []
+        summaries = self.env['mail.activity.schedule.summary']
         for template in templates:
             activity_type = template.activity_type_id
-            summary_line = activity_type.name
-            if template.summary:
-                summary_line += f": {template.summary}"
-            # We don't display deadlines when the user doesn't specify a plan_date
+            summary_line = summaries.new({'schedule_id': self})
```

Comment:
Try to use more standard way of creating embedded 2many. We generally create dict values, then we use commands on the o2m field. I agree your code is a bit more "object oriented" but it is less standard compared to odoo code :) something like
```
    line_values = []
    line_values.append({
        'summary': template.summary,
        'user_id': template.responsible_id.id,
    })
    ...
    return line_values
``